### PR TITLE
Enforce certain CSS properties to use CSS variables (stylelint-declaration-strict-value)

### LINKED ISSUE #11: Enforce certain props to use css variables
color, background-color and other stuff should use variables.

See: https://github.com/AndyOGo/stylelint-declaration-strict-value

### PR DESCRIPTION
**Pull request by jdiogopeixoto**
Added stylelint-declaration-strict-value plugin to enforce the following properties to use variables:
- any property with "color";
- font-family and font-weight;
- z-index;

Closes #11

### CODE REVIEW

File: rules/declaration-strict-value.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+// See https://github.com/AndyOGo/stylelint-declaration-strict-value
+
+const properties = [
+    '/color/',
+    'font-family',
+    'font-weight',
+    'z-index',
```
Comment (nlfonseca) [inline rules/declaration-strict-value.js]: I think that  we also can add  `fill`and `stroke` to the properties that is used to color `svg`

File: rules/declaration-strict-value.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+];
+
+module.exports = {
+    'plugins': [
+        'stylelint-declaration-strict-value',
+    ],
+    'rules': {
+        'scale-unlimited/declaration-strict-value': [properties, {}],
```
Comment (nlfonseca) [inline rules/declaration-strict-value.js]: we need to ignore the enforce of the variables in certain cases like `currentColor`, `transparent`, `inherit`.

File: rules/declaration-strict-value.js (hunk 1 above)
Comment (jdiogopeixoto) [reply rules/declaration-strict-value.js]: Added `fill` and `stroke` properties.

File: rules/declaration-strict-value.js (hunk 2 above)
Comment (jdiogopeixoto) [reply rules/declaration-strict-value.js]: Added an option to ignore `currentColor`, `transparent` and `inherit`.

**Comment by nlfonseca** [comment]
Good job 👍 

Normally the commit message is in the present tense.
Another note, before the merge we squash all commits.

Besides that, all good 👍

**Pull request merged**